### PR TITLE
Fix links and headings in v3 (unreleased) changelog

### PR DESCRIPTION
**Pull request by m-green**
This pull requests makes some minor fixes to the v3 (unreleased) changelog.

- Increase all heading levels by one, to avoid the duplicate h2 at the top.
- Fix broken links to docs inside the repo.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -113,15 +113,15 @@ Pull requests:
 - [#1458: Namespace nunjucks and components](https://github.com/alphagov/govuk-frontend/pull/1458)
 - [#1467: Update the main entry point in package.json](https://github.com/alphagov/govuk-frontend/pull/1467)
 
-### Migrate to the new accessible focus states
+#### Migrate to the new accessible focus states
 
 The focus state of components now meets the new WCAG 2.1 level AA requirements.
 
-You must update your component’s focus state to make your design consistent with our new focus styles.
+You must [update your component’s focus state](https://design-system.service.gov.uk/get-started/TODO) to make your design consistent with our new focus styles.
```
Comment (m-green) [inline CHANGELOG.md]: Update with link before merging.

File: CHANGELOG.md [hunk 2]
```diff
@@ -113,15 +113,15 @@ Pull requests:
...
 
 If you've extended or created components, you can no longer use the `govuk-focusable` or `govuk-focusable-fill` mixins in your Sass files.
 
-If you're using `govuk-focusable`, you must remove it. There’s no direct replacement, so you must use our Sass variables to make your components consistent with GOV.UK Frontend.
+If you're using `govuk-focusable`, you must remove it. There’s no direct replacement, so you must [use our Sass variables to make your components consistent](https://design-system.service.gov.uk/get-started/TODO#TODO) with GOV.UK Frontend.
```
Comment (m-green) [inline CHANGELOG.md]: Update with link before merging.

File: CHANGELOG.md [hunk 3]
```diff
@@ -180,24 +180,26 @@ We've also changed the background of the following components:
 - confirmation panels - `green` instead of `turquoise`
 - links in their hover state - `dark-blue` instead of `light-blue`
 
-If you're using legacy projects like GOV.UK Elements, you can keep your current colours by [turning on compatibility mode](https://github.com/alphagov/govuk-frontend/docs/installation/compatibility.md).
+If you're using legacy projects like GOV.UK Elements, you can keep your current colours by [turning on compatibility mode](https://github.com/alphagov/govuk-frontend/blob/master/docs/installation/compatibility.md).
+
+Read our [blog post about why we changed the colour palette](https://designnotes.blog.gov.uk/TODO).
```
Comment (m-green) [inline CHANGELOG.md]: Update with link before merging.

**Comment by nickcolley** [comment]
We'll want to manually update the release note with these changes too...

**Pull request merged**